### PR TITLE
Profile

### PR DESCRIPTION
**Pull request by OlivierLi**
- Child processes are now correctly followed
- Choose between wakeups and cpu_time profiling!
- Execution waits for dtrace to complete symbolizing after scenario has run.
- Add more instruction to README.md
- dtrace output is sent to a log file since most of the time it's of little interest.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,60 @@
...
+* Reboot
+
+=======
+=======
 
 ## benchmark.py
 Use to execute different usage scenarios and measure their power use using powermetrics.
+```
+./benchmakrk.py --measure ./results
+./benchmark.py profile --profile_mode cpu_time --chromium_executable=./bin/Chromium.app
```
Comment (eti-p-doray) [inline README.md]: Is "profile" the output dir? Let's put "./profile" for consistency

File: benchmark.py [hunk 2]
```diff
@@ -105,32 +111,41 @@ def Profile(scenario_config, output_dir, dry_run=False):
 
   subprocess_to_pid = {}
```
Comment (eti-p-doray) [inline benchmark.py]: Sounds like subprocess_to_pid is rather pid_to_subprocess

File: benchmark.py [hunk 3]
```diff
@@ -150,48 +165,55 @@ def main():
...
   if args.run_measure and args.dry_run:
       print("Dry running measure is not implemented !")
       exit(-1)
 
-  if args.run_profile and args.run_measure:
+  if args.profile_mode and args.run_measure:
       print("Cannot measure and profile at the same time, choose one.")
       exit(-1)
+  
+  if args.profile_mode and args.profile_mode not in ["wakeups", "cpu_time"]:
```
Comment (eti-p-doray) [inline benchmark.py]: Argparse has a "choices" argument that can do this, and is included in help option.

File: README.md (hunk 1 above)
Comment (OlivierLi) [reply README.md]: Sure! I fixed some problems with the other command too.

File: benchmark.py (hunk 2 above)
Comment (OlivierLi) [reply benchmark.py]: Good point!

File: benchmark.py (hunk 3 above)
Comment (OlivierLi) [reply benchmark.py]: Cool! Less code :)

File: benchmark.py (hunk 2 above)
Comment (OlivierLi) [reply benchmark.py]: It does! Done.

**Review by eti-p-doray — APPROVED** [review]

**Pull request merged**